library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -66,10 +66,10 @@ void device_strided_batched_matrix_copy(const void* src,
             size_t matrix_size = n1 * n2 * elem_size;
```

Comment:
This file's changes only consist of removing trailing whitespace and fixing typos